tests/TestKernel.php

```diff
@@ -53,17 +55,52 @@ function (ContainerBuilder $container) {
                     'symfonycasts.messenger_monitor.storage.doctrine_connection'
                 )->setPublic(true);
 
+                $container->setAlias('test.messenger.bus.default', 'messenger.bus.default')->setPublic(true);
+                $container->setAlias('test.messenger.receiver_locator', 'messenger.receiver_locator')->setPublic(true);
+
+                $messageHandlerDefinition = new Definition(TestableMessageHandler::class);
+                $messageHandlerDefinition->addTag('messenger.message_handler');
+                $container->setDefinition('test.message_handler', $messageHandlerDefinition);
+
                 $container->setParameter('kernel.secret', 123);
                 $container->prependExtensionConfig(
                     'framework',
                     [
-                        'session' => ['enabled' => true],
+                        'session' => [
+                            'enabled' => true,
+                            'storage_id' => 'session.storage.mock_file'
+                        ],
                         'router' => [
                             'resource' => 'kernel::loadRoutes',
+                            'type' => 'service',
                             'enabled' => true,
                         ],
+                        'messenger' => [
+                            'failure_transport' => 'failed',
+                            'transports' => [
+                                'queue' => [
+                                    'dsn' => 'doctrine://default?queue_name=queue',
+                                    'retry_strategy' => ['max_retries' => 0]
+                                ],
+                                'failed' => 'doctrine://default?queue_name=failed',
+                            ],
+                            'routing' => [
+                                TestableMessage::class => 'queue'
+                            ]
+                        ],
+                        'test' => true,
                     ]
                 );
+
+                if (!$container->hasDefinition('kernel')) {
+                    $container->register('kernel', static::class)
+                        ->setSynthetic(true)
+                        ->setPublic(true);
+                }
```

Comment:
Is this needed?

Reply:
yep otherwise i got this error:
```
11) SymfonyCasts\MessengerMonitorBundle\Tests\IntegrationTests\Storage\DoctrineConnectionTest::testSaveAndLoadMessage
Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException: You have requested a non-existent service "kernel".

/home/niko/works/messenger-monitor-bundle/vendor/symfony/dependency-injection/ContainerBuilder.php:980
/home/niko/works/messenger-monitor-bundle/tests/TestKernel.php:104
/home/niko/works/messenger-monitor-bundle/vendor/symfony/dependency-injection/Loader/ClosureLoader.php:38
/home/niko/works/messenger-monitor-bundle/vendor/symfony/config/Loader/DelegatingLoader.php:40
/home/niko/works/messenger-monitor-bundle/tests/TestKernel.php:122
/home/niko/works/messenger-monitor-bundle/vendor/symfony/http-kernel/Kernel.php:638
/home/niko/works/messenger-monitor-bundle/vendor/symfony/http-kernel/Kernel.php:544
/home/niko/works/messenger-monitor-bundle/vendor/symfony/http-kernel/Kernel.php:126
/home/niko/works/messenger-monitor-bundle/vendor/symfony/framework-bundle/Test/KernelTestCase.php:78
/home/niko/works/messenger-monitor-bundle/tests/IntegrationTests/AbstractDoctrineIntegrationTests.php:26
```

Reply:
hum ok, with thje `MicroKernelTrait` this is no more needed